template/package.json

```diff
@@ -43,7 +43,7 @@
   {{#if_eq builder 'builder'}}
   "build": {
     "productName": "{{ name }}",
-    "appId": "org.simulatedgreg.electron-vue",
+    "appId": "com.example.{{ name }}",
```

Comment:
May I suggest changing the appId line in the package.json template to include the author name in addition to the project name? 

    "appId": "com.{{ author }}.{{ name }}",

Reply:
Oh wait, I just remembered you can include additional info like email in the author name... In which case you'd have to filter the input string.